public/index.html

```diff
@@ -19,11 +20,16 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
+    <link rel="stylesheet" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/semantic-ui/2.3.1/semantic.min.css">
+    <script src="https://code.jquery.com/jquery-3.1.1.min.js" integrity="sha256-hVVnYaiADRTO2PzUGmuLJr8BLUSjGIZsDYGmIJLv2b8="
+        crossorigin="anonymous"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/semantic-ui/2.3.1/semantic.min.js"></script>
```

Comment:
Let's remove these script tags because they can be replaced by adding import statements in the App component.

For example:
`import 'semantic-ui-css/semantic.min.css';`

Reply:
The same should be done for jQuery.

Reply:
Semantic-ui added to package.json for remove script tag of semantic.min.js.